packages/edit-site/src/components/sidebar-navigation-screen/index.js

```diff
@@ -40,7 +42,7 @@ export default function SidebarNavigationScreen( {
 	const theme = getTheme( currentlyPreviewingTheme() );
 
 	return (
-		<VStack spacing={ 2 }>
+		<VStack spacing={ 0 }>
```

Comment:
This affects many screen, right? Just to confirm we want the design updated in this component.

Reply:
I believe this is intentional yes. 

cc @SaxonF